app/.env.dist

```diff
@@ -22,6 +22,12 @@ VITE_NOTEBOOK_LIST_TYPE=tabs
 VITE_NOTEBOOK_NAME=survey
 # What is the name of the app - used for App store and also as default heading
 VITE_APP_NAME=Fieldmark
+# App ID is the App identifier for the mobile builds, should be a reverse domain name
+VITE_APP_ID=org.fedarch.faims3
+# Bundle identifier can be the same as APP_ID but is used for app signing
+VITE_APPLE_BUNDLE_IDENTIFIER=au.edu.faims.electronicfieldnotebook
+# Team ID from the App Store, see IOS_Deployment docs
+VITE_APP_STORE_CONNECT_TEAM_ID=XXXXXXXXX
```

Comment:
These have suitable behaviour for local dev?

Reply:
Yes, I've now added some notes on local dev to the main README.md